benches/writebatch/bench_writebatch.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use test::Bencher;
+use tempdir::TempDir;
+use rocksdb::{DB, WriteBatch, Writable};
+
+fn writebatch(db: &DB, round: usize, batch_keys: usize) {
+    let v = b"operators are syntactic sugar for calls to methods of built-in traits";
+    for r in 0..round {
+        let batch = WriteBatch::new();
+        for i in 0..batch_keys {
+            let k = format!("key_round{}_key{}", r, i);
```

Comment:
no, actually we just need to get the relative tendency, not absolutely. but i still bench it, and the affection is less than 5%.